projects/angular-calendar/src/modules/common/calendar-a11y.interface.ts

```diff
@@ -0,0 +1,66 @@
+import { MonthViewDay, CalendarEvent } from 'calendar-utils';
```

Comment:
Let's remove this interface to keep things simpler, the reason for the `CalendarDateFormatterInterface` is because there are multiple different implementations of it provided, for this new a11y class the dev flow will probably be to just extend the built in one and override the defaults as required, rather than creating a new class from scratch.